features/HTTP.feature

```diff
@@ -484,9 +484,44 @@ Feature: Test HTTP API
     And I create a new role "role2" with id "role2"
     And I create a new profile "profile2" with id "profile2"
     And I create a user "user2" with id "user2-id"
-    When I log in as user2-id:testpwd2 expiring in 1h
+    When I log in as user2:testpwd2 expiring in 1h
     Then I'm able to find my rights
 
+  @usingHttp @cleanSecurity
+  Scenario: user credentials crudl
+    Given I create a user "nocredentialuser" with id "nocredentialuser-id"
+    Then I validate local credentials of user nocredentialuser with id nocredentialuser-id
+    Then I create local credentials of user nocredentialuser with id nocredentialuser-id
+    Then I check if local credentials exist for user nocredentialuser with id nocredentialuser-id
+    Then I get local credentials of user nocredentialuser with id nocredentialuser-id
+    Then I get local credentials of user nocredentialuser by id nocredentialuser
+    Then I log in as nocredentialuser:testpwd1 expiring in 1h
+    Then I log out
+    Then I update local credentials password to "testpwd2" for user with id nocredentialuser-id
```

Comment:
niptpicking: add a line to check that I cannot login with "testpwd1" anymore after changing the password.